src/z3c/rml/attr.py

```diff
@@ -307,7 +317,7 @@ class File(Text):
     The value itself can eith be be a relative or absolute path. Additionally
     the following syntax is supported: [path.to.python.mpackage]/path/to/file
     """
-    open = staticmethod(urllib.urlopen)
+    open = staticmethod(six.moves.urllib.request.urlopen)
```

Comment:
It looks like urllib supports data URIs but urllib2 doesn't, and six maps urlopen to urllib2.